### ISSUE TITLE
About the car val AP

### ISSUE THREAD

**Issue by ClacYc** [monster-ghost/MonoDistill#5]
Hello, Zhiyu.  Thanks for your awesome work!
Sorry to bother you!
Recently I try to reproduce the **car val AP results**,but I find the results AP of **trained .pth by myself** are **lower than** **results reported** , which is evaluated with provided checkpoint .pth . 
I am a little bit confused about this.Because even though I have used the completely --cfg parameters provided in paper(except that I used batch_size == 64 and trained on 8 Gpus ), I could not obtain the same results reported . 
Here are some comparisons:
**3D@IoU:0.7 ///: Mod.    Easy    Hard** 
**Provided checkpoint ///:** 18.47      24.31     15.76
**Trained by myself ///:** 17.78      23.54     15.16
Could you please know the reason behind it ? 
Thank you !

**Comment by monster-ghost** [comment]
Thank you for your attention. I think it is normal because mono3D performance is always unstable(about 1AP). Maybe you can train many times. Also we are also exploring some methods which makes results stable. 
By the way, If you use a larger batchsize, the learning rate needs to be adjusted accordingly

**Comment by ClacYc** [comment]
Got it. Thanks a lot.
I am looking forward to your modified repo anyway!
